azurerm/resource_arm_container_group.go

```diff
@@ -494,6 +518,17 @@ func resourceArmContainerGroupCreate(d *schema.ResourceData, meta interface{}) e
 	return resourceArmContainerGroupRead(d, meta)
 }
 
+// https://docs.microsoft.com/en-us/azure/container-instances/container-instances-vnet#virtual-network-deployment-limitations
+// https://docs.microsoft.com/en-us/azure/container-instances/container-instances-vnet#preview-limitations
+func checkVirtualNetworkCompatibility(d *schema.ResourceDiff, _ interface{}) (err error) {
```

Comment:
Done. My initial reasoning for doing the CustomizeDiff function was to avoid a situation where the user would try to move Windows containers to a virtual network leading to a situation where the old containers would be deleted but new ones could not be provisioned.